### PR TITLE
make the website beautiful

### PR DESCRIPTION
**Pull request by Powerlated**
Some changes:
- Fulfill #1 
- Somewhat redesign the entire website
- Get rid of redundant HTML and load it with  jquery from a templates folder
- Merge the about and home page

# Old website:
https://cdn.discordapp.com/attachments/438787210360258562/438841096471314442/unknown.png
# New website:
https://cdn.discordapp.com/attachments/438787210360258562/438840971287855119/unknown.png
https://powerlated.net/gmvcsc.github.io/

Same content, new looks.

### CODE REVIEW

File: calendar.html [hunk 1]
```diff
@@ -1,79 +1,72 @@
...
-          <a href="faq.html">FAQ</a> |
-          <a href="contact.html">Contact</a>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
-          created by:&nbsp; Chris Nelson &amp; Patrick Yang </p>
-      </div>
+    <div id="navigation"></div>
+
+    <div id="content">
+      <a href="events.html" Events-Registartion</a>
+        <h2>Calendar of Events</h2>
+        <p>
```
Comment (dwcaraway) [inline calendar.html]: Notice any tags here that might be unnecessary?

File: calendar.html [hunk 2]
```diff
@@ -1,79 +1,72 @@
...
+<head>
+  <meta content="text/html; charset=utf-8" http-equiv="Content-Type" />
+  <title>GMVCSC | Calendar of Events</title>
+  <link href="styles/style1.css" media="screen" rel="stylesheet" title="CSS" type="text/css" />
+  <style type="text/css">
+    .auto-style1 {
+      margin-left: 0px;
+    }
+  </style>
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
```
Comment (dwcaraway) [inline calendar.html]: Nice 🎉  working on those `jquery` skills here. I like that you're DRYing up the code by moving redundant sections into reusable blocks. that's a good practice.

**Review by dwcaraway — APPROVED** [review]
I like where you're going with it. It's a smart move to start removing repeating code early on, as it will make the site more maintainable. There's some areas in the HTML that could be replaced by CSS...maybe some stray `<br/>` 
 `&nbsp;` and white space characters. Cleaning some of this might make for a nice follow-on pull request. I'm good with the merge though. Thanks for working on the site.

**Comment by Powerlated** [comment]
That last commit I just made 'Update index.html' was just to revert a merge conflict misfix.

**Comment by Powerlated** [comment]
Okay, just added a commit that added some good stuff:
- It is now a single page site
- Load page contents under the navbar from a `content-pages` folder with `jquery`
- Add a hashtag to the URL corresponding to the content page loaded from the `content-pages`
folder.
- Add an event-registration page
- Center the footer

**Pull request closed without merging**